src/neynar-api/v1/openapi/apis/notifications-api.ts

```diff
@@ -34,17 +34,16 @@ import { ReactionsAndRecastsResponse } from '../models';
 export const NotificationsApiAxiosParamCreator = function (configuration?: Configuration) {
     return {
         /**
-         * Gets a list of mentions and replies to the user’s casts in reverse chronological order
+         * Gets a list of 15 mentions and replies to the user’s casts in reverse chronological order
          * @summary Get mentions and replies
          * @param {string} apiKey API key required for authentication.
          * @param {number} fid fid of a user
          * @param {number} [viewerFid] fid of the user viewing this information, needed for contextual information.
-         * @param {number} [limit] Number of results to retrieve (default 25, max 150)
          * @param {string} [cursor] Pagination cursor.
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        mentionsAndReplies: async (apiKey: string, fid: number, viewerFid?: number, limit?: number, cursor?: string, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
+        mentionsAndReplies: async (apiKey: string, fid: number, viewerFid?: number, cursor?: string, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
This change is not a part of PR https://github.com/neynarxyz/OAS/pull/142/files

It might have came in from different PR, and we nvr addressed this change

But, the limit is not being passed to `mentionsAndReplies` anymore.
Please update the wrapper method `fetchMentionAndReplyNotifications` in `src/neynar-api/v1/client.ts` and `src/neynar-api/neynar-api-client.ts`.

Also please update the respective examples in the JSDocs.

Reply:
Added